Resources/Prototypes/_Goobstation/Catalog/Fills/Belt/belts.yml

```
@@ -119,6 +119,18 @@
+- type: entity
+  id: ClothingBeltHolsterFilledChaplain
+  parent: ClothingBeltHolster
+  name: Chaplains' Holster
+  description: A shoulder holster from the Gods. Contains a set of dual pistols.
+  suffix: Filled, Chaplain
+  components:
+  - type: StorageFill
+    contents:
+    - id: WeaponPistolCasul
+    - id: WeaponPistolJackal
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)